src/net/connect.cpp

```diff
@@ -110,11 +111,11 @@ void resolve_hostname(
             }
             cb(*result);
         }, {}, poller);
```

Comment:
Shouldn't this `{}` become `settings`? Or is this something to be done in another pull request?